docs/guides/migrating-from-quantum-engine.rst

```diff
@@ -0,0 +1,264 @@
+=============================
+Migrating From Quantum Engine
+=============================
+
+By August 2023, Quantum Engine (QE) is being sun set.
+Running workflows remotely will only work with Compute Engine (CE).
+This document covers the differences between QE and CE, and how to move to CE.
+
+
+Am I Affected?
+==============
+
+If you're using ``orq login`` or ``orq login --qe``, you're on QE. Read this guide.
```

Comment:
Question: Isnt orq login default to CE since like 2 releases already?

Reply:
Thanks, fixed